packages/eyes/src/eyes-test/eyes.ts

```diff
@@ -0,0 +1,169 @@
+const {TestFailedError, TestResults} = require('@applitools/eyes.sdk.core');
+const {makeVisualGridClient, initConfig} = require('@applitools/visual-grid-client');
+const domNodesToCdt = require('@applitools/visual-grid-client/src/browser-util/domNodesToCdt');
+import path from 'path';
+import fs from 'fs';
+import glob from 'glob';
+import chalk from 'chalk';
+import {JSDOM} from 'jsdom';
+import {consoleLog} from '@ui-autotools/utils';
+import {parseSnapshotFilename} from '../generate-snapshots/filename-utils';
+
+interface ITestResult {
+  status: 'error' | 'new' | 'modified' | 'unmodified';
+  url?: string;
+  error?: any;
+}
+
+function getGridClientConfig(projectPath: string) {
+  const projectName = require(path.join(projectPath, 'package.json')).name;
+  if (!projectName) {
+    throw new Error('The project should have a package.json file with a "name" field.');
+  }
+
+  if (!process.env.APPLITOOLS_API_KEY) {
+    throw new Error('The environment variable "APPLITOOLS_API_KEY" needs to be defined.');
+  }
+
+  const branchName = projectName + '/master';
+  const viewportWidth = 800;
+  const viewportHeight = 600;
+
+  return {
+    appName: projectName,
+    branchName,
+    batchName: projectName,
+    browser: {
+      name: 'chrome',
+      width: viewportWidth,
+      height: viewportHeight
+    }
+  };
+}
+
+interface IResource {
+  url: string;
+  type: string;
+  value: Buffer;
+}
+
+function getStaticResources(cssFilenames: string[], resourceDir: string): {[url: string]: IResource} {
+  const resources: {[url: string]: IResource} = {};
+  for (const cssFilename of cssFilenames) {
+    resources[cssFilename] = {
+      url: cssFilename,
+      type: 'text/css',
+      value: fs.readFileSync(path.join(resourceDir, cssFilename))
+    };
+  }
+  return resources;
+}
+
+function formatName(filename: string) {
+  const {compName, simName, styleName} = parseSnapshotFilename(filename, '.snapshot.html');
+
+  return `${compName}: ${simName}. Style: ${styleName}.`;
```

Comment:
If we're using the base style we need to return simply `${compName}: ${simName}.`